package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "ckeditor4-dev",
-	"version": "4.14.2",
+	"version": "4.14.3",
```

Comment:
Don't forget to update the `lock` file.

Reply:
I think we shouldn't change the version because it is related to real version of CKeditor. It is just fork repo with out addons. And the latest update was done by @andrey-ponamarev in  https://github.com/oat-sa/ckeditor-dev/pull/16
I suppose we need revert version to 4.14.1